packages/@aws-cdk/cdk/lib/core/tokens.ts

```diff
@@ -67,7 +88,7 @@ export class Token {
         }
 
```

Comment:
Not necessarily, since it's not unlikely that people will lazily return literals. I don't want to fail on that case. It will fail to properly join later on, though.